api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java

```diff
@@ -41,7 +41,10 @@
 
     private static final long serialVersionUID = 1L;
 
-    private int replicas;
+    private static final int DEFAULT_REPLICAS = 3;
+
+    private int replicas = DEFAULT_REPLICAS;
```

Comment:
Should I change `protected static final int DEFAULT_REPLICAS = 3;` into public in `KafkaMirrorMakerCluster` class, and use it here, just to avoid having the default value defined multiple places ?

Reply:
I think it can be probably completely removed from `KafkaMirrorMakerCluster`, or? Or is it still needed for something there?

Reply:
Is it still needed for something? I think since it cannot be undefined you should not need it anymore in `KafkaMirrorMakerCluster`, or?

Reply:
It is used only here: https://github.com/strimzi/strimzi-kafka-operator/blob/master/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerCluster.java#L120

And I think this can be removed.